daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1551,12 +1551,12 @@ func (devices *DeviceSet) GetDeviceStatus(hash string) (*DevStatus, error) {
 
 	if sizeInSectors, mappedSectors, highestMappedSector, err := devices.deviceStatus(info.DevName()); err != nil {
```

Comment:
This not compiling :) You need to split assignment and `if err != nil` to separate lines.